packages/create-block/README.md

```diff
@@ -1,44 +1,69 @@
+- [Usage](#usage)
+    - [`slug`](#slug)
+    - [Interactive Mode](#interactive-mode)
+    - [`options`](#options)
+- [Available Commands](#available-commands)
+- [External Project Templates](#external-project-templates)
+- [Contributing to this package](#contributing-to-this-package)
 
-## Quick start
 
-You only need to provide the `slug` – the target location for scaffolded plugin files and the internal block name.
+## Quick start
 
 ```bash
 $ npx @wordpress/create-block todo-list
 $ cd todo-list
 $ npm start
 ```
 
+The `slug` provided (`todo-list` in the example) defines the folder name for the scaffolded plugin and the internal block name. The WordPress plugin generated need to [be installed manually](https://wordpress.org/support/article/managing-plugins/#manual-plugin-installation).
+
+
 _(requires `node` version `14.0.0` or above, and `npm` version `6.14.4` or above)_
 
-It creates a WordPress plugin that you need to [install manually](https://wordpress.org/support/article/managing-plugins/#manual-plugin-installation).
 
-[Watch a video introduction to create-block on Learn.wordpress.org](https://learn.wordpress.org/tutorial/using-the-create-block-tool/)
+> [Watch a video introduction to create-block on Learn.wordpress.org](https://learn.wordpress.org/tutorial/using-the-create-block-tool/)
 
 ## Usage
 
-The following command generates a project with PHP, JS, and CSS code for registering a block with a WordPress plugin.
+The `create-block` command generates a project with PHP, JS, and CSS code for registering a block with a WordPress plugin.
 
 ```bash
 $ npx @wordpress/create-block [options] [slug]
 ```
 
 ![Demo](https://user-images.githubusercontent.com/699132/103872910-4de15f00-50cf-11eb-8c74-67ca91a8c1a4.gif)
 
-`[slug]` is optional. When provided, it triggers the quick mode where it is used as the block slug used for its identification, the output location for scaffolded files, and the name of the WordPress plugin. The rest of the configuration is set to all default values unless overridden with some options listed below.
+> When you scaffold a block, you must provide at least a `slug` name, the `namespace` which usually corresponds to either the `theme` or `plugin` name. In most cases, we recommended pairing blocks with WordPress plugins rather than themes, because only using plugin ensures that all blocks still work when your theme changes.
+
+
+### `slug`
+
+The use of `[slug]` is optional. 
+
+When provided, it triggers the _quick mode_ where it is used:
+- as the block slug (required for its identification)
+- as the output location (folder name) for scaffolded files
+- as the name of the WordPress plugin. 
+
+The rest of the configuration is set to all default values unless overridden with some options listed below.
+
+### Interactive Mode
+
+When no `[slug]` or `[options]` are provided, the script will run in interactive mode giving a chance to customize the important options before generating the files.
```

Comment:
Interactive mode is only triggered if no slug is passed. Otherwise, it will use defaults.

Reply:
Fixed and improved ✅